src/mockServiceWorker.js

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable */
```

Comment:
In case it isn't obvious, this file is programmatically generated by MSW and doesn't need to be reviewed. ;)